extensions/amp-fit-text/0.1/amp-fit-text.css

```diff
@@ -17,17 +17,38 @@
 .i-amphtml-fit-text-content,
 .i-amphtml-fit-text-content.i-amphtml-fill-content {
   display: block;
-  /* @alternative */ display: flex;
+  /* @alternative */
+  display: flex;
   flex-direction: column;
   flex-wrap: nowrap;
   justify-content: center;
 }
 
+.i-amphtml-fit-text-content {
+  z-index: 2;
+  visibility: hidden; /* overriden to visible in layoutCallback */
+}
+
+.i-amphtml-fit-text-content-wrapper {
```

Comment:
A comment here would be good. Know it was the default before, but won't remember in a few months.

Reply:
Done

Reply:
I've added a comment indicating that it must be kept in sync with the JS file.
Not sure what you mean by default since I don't think it ever changes!